tests_integration/nativeapp/test_teardown.py

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import os
+from shlex import split
```

Comment:
Any reason to use this over `split(' ')` for this particular test or just wanting to implement a cleaner pattern?

Reply:
No specific reason, just following this pattern since [this comment](https://github.com/snowflakedb/snowflake-cli/pull/1461#discussion_r1729204213)